_posts/blog/2017-07-20-upday_blogathon.md

```diff
@@ -0,0 +1,113 @@
+
+This needed to change. **But how can one convince people to invest time and energy into something they don’t really like or want to do?**
+
+## How?
+
+At upday, from time to time we organise **Hackathons**. During a Hackathon, people work intensely in unusual team formations 
+(often business people also participate), test new technologies and dive into certain topics in order to produce something valuable, 
+interesting or surprising within only a few hours of work. A few of these results have actually found their way into our product. 
+Others were just cool. And some, of course, just sucked.
+
+Our plan now was to “abuse” one of those Hackathons to generate blog posts. The idea of a **Blogathon** was born. 
+Okay, I know what you might be thinking right now.
+
+Anyway. Presenting this idea provoked mixed reactions. Everything from “cool” to “it sucks big time”. 
+But no risk no fun, it’s only painful at the start and attending was optional anyway.
+
+### Timetable
+
+* we announced the Blogathon two weeks in advance
+* one week before the Blogathon, we set up an **Ideas Board** that served two purposes:
+    * to let people start thinking about possible topics
+    * to collect topics and trigger the creativity of others
+* we had the Blogathon on a Thursday
+
+<left>
+    <picture>
+        <img src="/images/blog/blogathon_2017/blogathon_2017_ideas_board.jpg" 
+        width="50%" height="50%"
+        alt="Ideas Board" >
+    </picture>
+</left>
+
+### Agenda
+
+<left>
+    <picture>
+        <img src="/images/blog/blogathon_2017/blogathon_2017_agenda.jpg" 
+        width="50%" height="50%"
+        alt="Agenda" >
+    </picture>
+</left>
+
+## And...?
+
+To make a long story short, our Blogathon was quite a success. **We ended up with 8 finished or nearly finished articles** covering topics 
+from tutorials to personal insights from our daily business. You will be able to read the best of them in our [upday tech blog](http://upday.github.io/) 
+during the next few weeks.
+
+To finally **publish** them, all articles have to pass a **quality gate**. 
+We are used to challenging ourselves in terms of high standards so why shouldn’t this apply to our written output as well? 
+
+These are the finishing steps to get a blog post out on the street:
+
+* **finish** your article to the point you are happy and proud of it
+* let one or two **peers review** it for clarity (this might be an iterative process)
+* let a **native speaker review** it
+* clone and create a branch on our blog repository
+* add your article (Markdown) 
+* optimise the **layout**, give it a **readable structure**, **emphasise** important passages and add **keywords**
+* give your article a **final review**
+* merge branch and publish it
+
+## Conclusion
+
+**Everybody can write** and everybody should! Sharing thoughts or findings with others is rewarding. 
+It is more blessed to give than to receive. Writing also serves as a good reflection on what you have achieved. 
```

Comment:
I would suggest to be consistent with the contractions, I wouldn't use them except when you refer to your thoughts (to give them an informal touch) i.e. I'm convinced